CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # Recurly PHP Client Library CHANGELOG
 
+## Version 2.7.0 (unreleased)
+
+
+
 ## Version 2.6.0 (unreleased)
 
 * Added support for `original_transaction` to `Recurly_Transaction` [#238](https://github.com/recurly/recurly-client-php/pull/238)
 * Added `Recurly_AccountBalance` [#239](https://github.com/recurly/recurly-client-php/pull/239)
 * Print warnings when using a deprecated version of the API.
+* Allow credit adjustments (`Recurly_Adjustment`) to specify an `origin` of `external_gift_card`
```

Comment:
This needs a rebase to account for the changes to this file. I just got the 2.6.0 release out so let's put this in the 2.7.0 section where all the API V2.4 stuff will go.